File: Treinamento/Java/Estrutura_de_repetição/Questão11.java
import java.util.Scanner;
public class Questão11 {
    public static void main(String[] args) {
        int qtd_pares = 0;
        Scanner s;
        s = new Scanner(System.in);

        for (int c = 0; c < 10; c++) {
            System.out.print("Digite o número desejado: ");
            int n = s.nextInt();
            if (n % 2 == 0) {
                qtd_pares += 1;
            }
        }
        System.out.println("Quantidade de números pares: " + qtd_pares);
        System.out.println("Quantidade de números ímpares: " + (10 - qtd_pares));
    }
}
                

File: Treinamento/Python/Exercícios/Estrutura_de_repetição/Questão11.py
print('*** Números pares e ímpares')

n_pares = 0

for c in range(0, 10, +1):
    n = int(input('Digite o número desejado: '))
    if n % 2 == 0:
        n_pares += 1   

print('Números pares: ' + str(n_pares))
print('Números ímpares: ' + str(10 - n_pares))
        
